data/presets/natural/tree.json

```diff
@@ -1,14 +1,24 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
```

Comment:
Hello @tordans thanks for your comments. You are right genus, species and taxon are all in latin and reserved for user with some tree knowledge. But still, today genus and species are the only tag on trees that allows the user to identify a tree (Is is an Oak ? Is it a Pine ? ...) So for me, it is totally """mandatory""" that we add at least "genus" and "species".

You also have to think about the fact that some OpenData containing genus, species, start_date, ... have been imported to OSM in some cities. So the user can have the info about what kind of tree it is.

Reply:
I have to agree with @tordans here: These fields are too special for regular mappers: It cannot be expected that non experts know the required values. If we would display these fields by default, there is a relatively high probability that mappers would enter erroneous values like "oak" into the field(s).

Eventually it would be nice to have a special field in iD which lets people select the kind of tree from a list and the respective `species`/`taxon`/`genus` are filled in automatically. But that's something for another day.